internal/app/machined/pkg/controllers/k8s/extra_manifest.go

```diff
@@ -139,8 +139,22 @@ func (ctrl *ExtraManifestController) Run(ctx context.Context, r controller.Runti
 	}
 }
 
-func (ctrl *ExtraManifestController) download(ctx context.Context, r controller.Runtime, logger *log.Logger, manifest config.ExtraManifest) (id resource.ID, err error) {
-	id = fmt.Sprintf("%s-%s", manifest.Priority, manifest.URL)
+func (ctrl *ExtraManifestController) process(ctx context.Context, r controller.Runtime, logger *log.Logger, manifest config.ExtraManifest) (id resource.ID, err error) {
```

Comment:
I think that long function should become shorter, not longer:

```
func process(…) {
  id := …

  if if manifest.InlineManifest != "" {
    return ctrl.processInline(…)
  }

  return ctrl.processURL(…)
}
```

Reply:
thanks, fixed!